core/assertion.lisp

```diff
@@ -24,6 +25,11 @@
     (and (boundp debug-on-error-symbol)
 	 (symbol-value debug-on-error-symbol))))
 
+(defvar *quit-on-failure*
+  (let ((quit-on-failure-symbol (intern (string :*rove-quit-on-failure*) :cl-user)))
+    (and (boundp quit-on-failure-symbol)
+         (symbol-value quit-on-failure-symbol))))
```

Comment:
Wouldn't this be better done outside the rove?

Reply:
This is a way to configure **before** loading Rove.
It is already done for `*debug-on-error*`, so I suppose @Sasanidas refers to it.

You can set `(setf *rove-debug-on-error* t)` in `~/.roswell/init.lisp` without loading Rove whenever you run Lisp.

Reply:
Indeed.
I was thinking about the wildness of using symbols from the cl-user package.